docs/otlp-source.md

```diff
@@ -0,0 +1,41 @@
+# Using the OTLP source
+
+Historically, agents used by this Chart sent logs and metrics data to a [generic HTTP Source][http_source] in Sumo Logic. Ever since the
+[version 3][v3] release, the Chart predominantly uses the [OpenTelemetry Collector][otel], and consequently also the [OTLP protocol][otlp].
+The data needed to be converted to the formats the generic HTTP source supports before being sent.
+
+Recently, we've added support for directly sending data using the OTLP protocol to Sumo Logic. This is achieved using the [OTLP
+source][otlp_source]. This document explains how to use this new source with the Helm Chart.
+
+**Note:** OTLP Source is currently available for the following signal types: logs
```

Comment:
This should be rephrased. The OTLP Source is available for all signal types: logs, metrics, traces. It is our chart that currently only supports sending logs to an OTLP Source.

Reply:
Good point, will change this to be a statement about support in the Helm Chart.